File: Java Code/Cheer_Creator.java
import java.util.Scanner;

public class Cheer_Creator {
	
	public static void main(String[] args) {
		String sound="";
		Scanner input = new Scanner(System.in);
		int yard=input.nextInt();
		input.close();
		if (yard >10) {
			sound="High Five";
		} else if(yard <1) {
			sound="shh";
		} else {
			for (int i=1; i<=yard;i++) {
				sound += "Ra!";
			}
		};
		System.out.println(sound);

	}

}

File: Python/Cheer Creator.py
def sound(yards):
    if yards<1:
        return 'shh'
    elif yards>10:
        return 'High Five'
    else:
        return 'Ra!'*yards
try:
    moved_yard=int(input('Moved yard:'))
    print(sound(moved_yard))
except:
    print('Invalid number!')